libs/schema/src/index.ts

```diff
@@ -76,6 +76,12 @@ export interface Project {
   projectType: string;
   root: string;
   architect: Builder[];
+  recentActions: RecentAction[];
+}
+
+export interface RecentAction {
```

Comment:
Our graphql type generators should make this interface for you. Use those generated interfaces so they are kept inline with our schema.

Reply:
Ok